src/visibility.js

```diff
@@ -184,6 +164,11 @@ class Visibility extends BasePlugin {
     this.eventManager.listen(this.player, this.player.Event.LEAVE_PICTURE_IN_PICTURE, () => {
       this._isInPIP = false;
     });
+    this.player.viewabilityManager.observe(this._appTargetContainer, this.player.config.visibility.playerThreshold / 100, (visible: boolean) => {
```

Comment:
don't you want it to be able to config separately? I mean if the plugin doesn't have a config threshold it will take the player threshold the same for targetContainer?

Reply:
In general the entire reason to expose the visibility manager as a service is for any plugin to use it with its own thresholds, but in this case the floating plugin should have the exact player thresholds so we can avoid needing to set it on the plugin level and use the player.
We can use this maybe as an escape hatch for any future issue and do what @Yuvalke mentioned to use player defaults and only use plugin config if it was set by app.

Reply:
I don't understand - Do you want have the playerThreshold as default in the viewabilityManager and only if someone passes it in the observe to use it?

Reply:
exactly - specifically for floating - it needs to have exactly the same threshold as the player IIRC

Reply:
done